wlutil/install.py

```diff
@@ -21,6 +15,7 @@ def fullRel(base, target):
     return os.path.relpath(str(target), start=str(base))
 
 def installWorkload(cfgName, cfgs):
+    fsWork = ((Path(getCtx()['root-dir']) / "../../deploy/workloads").resolve())
```

Comment:
this should use getOpt instead of getCtx()[] (that is the consistent syntax that I've used elsewhere in the code).